dist/common/scripts/scylla_util.py

```diff
@@ -559,13 +792,18 @@ def is_gce():
     return gcp_instance.is_gce_instance()
 
 
+def is_azure():
+    return azure_instance.is_azure_instance()
+
 def get_cloud_instance():
     if is_ec2():
         return aws_instance()
     elif is_gce():
         return gcp_instance()
+    elif is_azure():
+        return azure_instance()
     else:
-        raise Exception("Unknown cloud provider! Only AWS/GCP supported.")
+        raise Exception("Unknown cloud provider! Only AWS/GCP and AZURE supported.")
```

Comment:
`Unknown cloud provider. Only AWS, GCP, and Azure are supported.`

Reply:
fixed